README.md

```diff
@@ -96,6 +96,32 @@ const orderId = await cowSdk.cowApi.sendOrder({
 console.log(`https://explorer.cow.fi/rinkeby/orders/${orderId}`)
 ```
 
+SDK also allows you to decode the appData hash to get the ipfs appData document
+
+```js
+  const hash = '0xa6c81f4ca727252a05b108f1742a07430f28d474d2a3492d8f325746824d22e5'
+  const appDataDoc  = await decodeAppData(hash)
```

Comment:
Do we plan to expose some utils directly from the NPM package, or will there be part of the SDK? 
This `decodeAppData` is not imported or anything in the example, we have to make sure the examples work.